provd/tests/test_phonebook.py

```diff
@@ -13,20 +13,20 @@ class TestAddXiVOPhonebookURL(unittest.TestCase):
     def setUp(self):
         self.raw_config = {
             u'X_xivo_phonebook_ip': u'8.8.8.8',
-            u'X_xivo_phonebook_profile': u'prof',
+            u'X_xivo_phonebook_profile': u'prof',  # This value is not used anymore
```

Comment:
why did you leave the key in the config? Does it break anything if its removed?

Reply:
No it does not break anything. I will remove it.